<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariablesPanelClass</class>
 <widget class="QDialog" name="VariablesPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VariablesPanel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="tbwVariables">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Variable name</string>
      </property>
      <property name="toolTip">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
      <property name="toolTip">
       <string>Filter files concerned by replacement</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="toolTip">
       <string>Remove variable</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pBtnAddVariable">
     <property name="toolTip">
      <string>Add new variable</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>498</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="toolTip">
      <string>Enable step</string>
     </property>
     <property name="text">
      <string>Enable step</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
